templates/start.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+    <form action="/start/" method="post">
```

Comment:
Но тут нужно убрать атрибут `action`, иначе будет ошибка `404`
```suggestion
    <form method="post">
```